src/Simplex/RemoteControl/Client.hs

```diff
@@ -76,6 +78,9 @@ xrcpBlockSize = 16384
 helloBlockSize :: Int
 helloBlockSize = 12288
 
+announceBlockSize :: Int
+announceBlockSize = 1024 -- TODO: put a real number
```

Comment:
what's the number? or 1024 is ok?

Reply:
something under MTU while covering announcement json,
1024 should be ok.